### DISCUSSION TITLE
Cache provisioning

### DISCUSSION THREAD

**Discussion by Vietdq-dev** [codemagic-ci-cd/codemagic-docs#1940] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I have updated the provisioning key and name is `Provisioning GEMA` to the file codemagic.yaml, but I still get the provisioning name old

### Expected results

I get an error when uploading to Appstore
<img width="993" alt="Screenshot 2023-08-01 at 13 41 54" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/80184186/e7f67fe2-b017-443c-b94a-792ea1093949">

### Actual results

Upload success with provisioning new

### Build id (optional)

64c8b13a304a399e7241af27

**Comment by Vietdq-dev** [comment]
Yes, The provisiong upload correct

Vào Th 5, 3 thg 8, 2023 vào lúc 15:51 Karl Kristjan Tamm <
***@***.***> đã viết:

> Hi @Vietdq-dev <https://github.com/Vietdq-dev>
>
> I'd like to get additional information: you are using automatic code
> signing and you have uploaded the new 'Provisioning GEMA' provisioning
> profile to Codemagic's iOS Signing Identities and it is combined with the
> right certificate?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/codemagic-ci-cd/discussions/1940#discussioncomment-6625363>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATDYG6XABPXVWI4YVEXSW5LXTNRAXANCNFSM6AAAAAA3CMDF2U>
> .
> You are receiving this because you were mentioned.Message ID:
> <codemagic-ci-cd/codemagic-docs/repo-discussions/1940/comments/6625363@
> github.com>
>


-- 
*[N]* : Đào Quốc Việt
*[E]* : ***@***.*** ***@***.***>
*[P]* : 0356.858.608
*[F]* : Facebook.com/Nhokquay9x26
*[S]* : College Iechnology Information

**Reply by kulakarla** [reply]
What kind of Code signing are you using? Are you using code signing identities (you have uploaded the certificate and provisioning profile), automatic code signing or manual code signing?

**Reply by Vietdq-dev** [reply]
@kulakarla Yes,I use automatic code signing

**Reply by kulakarla** [reply]
Hey @Vietdq-dev, did you get this issue fixed? I see you opened a new thread about publishing to App Store. I would like to close this and look at your other issue there

**Comment by Vietdq-dev** [comment]
@kulakarla Hello, Yes, I resolved it, could you close it

**Comment by kulakarla** [comment]
Closing this as the poster solved it themselves.
